vscode/microsoft-kiota/src/util.ts

```diff
@@ -161,3 +161,12 @@ export function allGenerationLanguagesToString(): string[] {
   let allSupportedLanguages: string[] = allGenerationLanguages.map(langEnum => KiotaGenerationLanguage[langEnum]);
   return allSupportedLanguages;
 }
+
+export function isValidUrl(url: string): boolean {
+  try {
+    new URL(url);
```

Comment:
Shall we add a check for HTTP URL here? What will mailto: URL do in this case? :-)

Like (probably better with a case insensitive check):
 return newUrl.protocol === 'http:' || newUrl.protocol === 'https:';

Reply:
new (url) validates the scheme too. for validity. So no need to. If scheme is not provided it returns valid url false. Just tested it.